boards/kernel_layout.ld

```diff
@@ -261,6 +292,13 @@ SECTIONS
         . = ALIGN(MPU_MIN_ALIGN);
         *(.app_memory)
     } > ram
+
+    /* Discard RISC-V relevant .eh_frame, we are not doing unwind on panic
```

Comment:
I'm guessing this proved necessary due to size? This isn't something you necessarily want to do as it was will make a debugger less useful; unwind info isn't just used for runtime panics

Reply:
I have no idea where that came from, probably rust-riscv I started from.